app/controllers/application_controller.rb

```diff
@@ -12,6 +12,6 @@ def switch_locale(&action)
   private
 
   def unauthorized
-    redirect_to root_url, :alert => "You have no access here"
+    redirect_to root_url, :alert => "<%= t('.no_access')%>"
```

Comment:
First you should use new syntax `alert: "You have no access here"`
Second `<%= %>` can be use only on **View** level so in all HTML's not in Ruby code.
The code should looks like: `alert: t('.no_access')`.